python-client/giskard/commands/cli_server.py

```diff
@@ -96,6 +96,30 @@ def get_container(version=None, quit_if_not_exists=True) -> Optional[Container]:
             return None
 
 
+def _backend_ready(endpoint) -> bool:
+    try:
+        response = requests.get(endpoint)
+
+        if response.status_code != 200:
+            return False
+
+        return "UP" == response.json()['components']['readinessState']['status']
+    except OSError:
```

Comment:
`requests` can throw other errors (`ConnectTimeout` for example), I think we can simply catch BaseException and return False if it happens. (make sure to handle `KeyboardInterrupt` separately and reraise `click.Abort()`)

In this case you can even replace 
```
        if response.status_code != 200:
            return False
```

with 

```
response.raise_for_status()
```

since it'll be caught by the same BaseException handler